src/components/DropdownOptions.tsx

```diff
@@ -9,6 +9,15 @@ const DropdownOptions = () => {
     history.push('/auth');
   };
 
+  const handleDashboard = () => {
+    if (localStorage.getItem('type') === 'volunteer') {
```

Comment:
this is making two calls to localstorage which is a lot more expnsive than just reading one variable.  save it in a variable on the first line of the function, then switch based on that.